quint/src/names/base.ts

```diff
@@ -49,7 +50,7 @@ export type LookupDefinition = (QuintDef | ({ kind: 'param' } & QuintLambdaParam
 /**
  * A module's definitions, indexed by name.
  */
-export type DefinitionsByName = Map<string, LookupDefinition & { hidden?: boolean }>
+export type DefinitionsByName = Map<string, (LookupDefinition & { hidden?: boolean })[]>
```

Comment:
Looks like `LookupDefinition` already has `hidden?`, so this type can be cleaner

```suggestion
export type DefinitionsByName = Map<string, LookupDefinition[]>
```

See https://github.com/informalsystems/quint/blob/f51cf1aa721caf907f03e45b53f44ccf01e25972/quint/src/names/base.ts#L28-L32

Reply:
Thanks! Much better